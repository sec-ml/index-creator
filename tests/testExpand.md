| term                  | sub-term              | notes          | book | page |
| --------------------- | --------------------- | -------------- | ---- | ---- |
| a && b && c && d && e | z && y && x && w && v | test expanding | 1    | 2    |
