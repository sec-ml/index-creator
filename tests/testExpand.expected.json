[
  {
    "term": "a",
    "subTerm": "v",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "a",
    "subTerm": "w",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "a",
    "subTerm": "x",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "a",
    "subTerm": "y",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "a",
    "subTerm": "z",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "b",
    "subTerm": "v",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "b",
    "subTerm": "w",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "b",
    "subTerm": "x",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "b",
    "subTerm": "y",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "b",
    "subTerm": "z",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "c",
    "subTerm": "v",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "c",
    "subTerm": "w",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "c",
    "subTerm": "x",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "c",
    "subTerm": "y",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "c",
    "subTerm": "z",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "d",
    "subTerm": "v",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "d",
    "subTerm": "w",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "d",
    "subTerm": "x",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "d",
    "subTerm": "y",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "d",
    "subTerm": "z",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "e",
    "subTerm": "v",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "e",
    "subTerm": "w",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "e",
    "subTerm": "x",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "e",
    "subTerm": "y",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  },
  {
    "term": "e",
    "subTerm": "z",
    "notes": "test expanding",
    "book": 1,
    "page": "2"
  }
]
